### PR TITLE
fix(docs): Fix feedback form UI

### LINKED ISSUE #4522: Fix feedback form UI

### PR DESCRIPTION
**Pull request by vivekjain23**
# Description of change

This PR bring up the changes to fix the feedback form UI.

## Links to any relevant issues

Fixes #4522  

## Type of change

- Documentation Fix

## How the change has been tested

Changes were built locally.

## Change checklist

- [X] I have followed the contribution guidelines for this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by lucas-tortora** [comment]
@vivekjain23 , I think you added the wrong for the issue

File: docs/site/src/components/FeedbackForm/index.tsx [hunk 1]
```diff
@@ -19,8 +19,7 @@ const FeedbackForm = () => {
   };
 
   return (
-    <div className="feedback-container">
-      <div className="divider"></div>
+    <div className="feedback-container bg-[#0000001a] dark:bg-[#1e1e1e] pt-1 px-8 pb-5 mt-4 rounded-lg">
```
Comment (lucas-tortora) [inline docs/site/src/components/FeedbackForm/index.tsx]: Could you add this to the component's stylesheet?

**Review by lucas-tortora — CHANGES_REQUESTED** [review]

**Comment by vivekjain23** [comment]
> @vivekjain23 , I think you added the wrong for the issue

Yup, my bad. Updated it just now.

File: docs/site/src/components/FeedbackForm/index.tsx (hunk 1 above)
Comment (vivekjain23) [reply docs/site/src/components/FeedbackForm/index.tsx]: Sure.

**Review by Ginowine — APPROVED** [review]

**Review by lucas-tortora — APPROVED** [review]

**Pull request merged**